From fba0aceab88b64e91bed982b738323ee7bab45c2 Mon Sep 17 00:00:00 2001
From: Peng Ma <peng.ma@nxp.com>
Date: Tue, 16 Oct 2018 07:34:12 +0000
Subject: [PATCH 060/741] dmaengine: fsl-dpaa2-qdma: the position of dpaa2-io.h
 is

commit 17dd0bdf415dda0b4b37d671e5fa8ed0e05b6425 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

changed, need to update include file

this patch is to fixed compile error for dpaa2-qdma.c file

Signed-off-by: Peng Ma <peng.ma@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/dma/dpaa2-qdma/dpaa2-qdma.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/dma/dpaa2-qdma/dpaa2-qdma.c b/drivers/dma/dpaa2-qdma/dpaa2-qdma.c
index c29eda063468..2c645a140612 100644
--- a/drivers/dma/dpaa2-qdma/dpaa2-qdma.c
+++ b/drivers/dma/dpaa2-qdma/dpaa2-qdma.c
@@ -35,8 +35,7 @@
 #include "../virt-dma.h"
 
 #include <linux/fsl/mc.h>
-#include "../../../drivers/staging/fsl-mc/include/dpaa2-io.h"
-#include "../../../drivers/staging/fsl-mc/include/dpaa2-fd.h"
+#include <soc/fsl/dpaa2-io.h>
 #include "fsl_dpdmai_cmd.h"
 #include "fsl_dpdmai.h"
 #include "dpaa2-qdma.h"
-- 
2.17.1

